x-pack/plugins/security_solution/public/assistant/get_comments/stream/message_text.tsx

```diff
@@ -0,0 +1,171 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import {
+  EuiMarkdownFormat,
+  EuiSpacer,
+  EuiText,
+  getDefaultEuiMarkdownParsingPlugins,
+  getDefaultEuiMarkdownProcessingPlugins,
+} from '@elastic/eui';
+import { css } from '@emotion/css';
+import classNames from 'classnames';
+import type { Code, InlineCode, Parent, Text } from 'mdast';
+import React from 'react';
+import type { Node } from 'unist';
+import { customCodeBlockLanguagePlugin } from '../custom_codeblock/custom_codeblock_markdown_plugin';
+import { CustomCodeBlock } from '../custom_codeblock/custom_code_block';
+
+interface Props {
+  content: string;
+  loading: boolean;
+}
+
+const ANIMATION_TIME = 1;
+
+const cursorCss = css`
+  @keyframes blink {
+    0% {
+      opacity: 0;
+    }
+    50% {
+      opacity: 1;
+    }
+    100% {
+      opacity: 0;
+    }
+  }
+
+  animation: blink ${ANIMATION_TIME}s infinite;
+  width: 10px;
+  height: 16px;
+  vertical-align: middle;
+  display: inline-block;
+  background: rgba(0, 0, 0, 0.25);
```

Comment:
We may want to use a theme variable here as the cursor is a bit hard to see in dark mode:

<img width="274" alt="image" src="https://github.com/elastic/kibana/assets/2946766/ff2acb66-b901-423e-91ff-409d7fae905f">